### DISCUSSION TITLE
[eCharts] I want to implement a new funnel chart. How should I proceed ?

### DISCUSSION THREAD

**Discussion by ValentinC-BR** [apache/superset#27301] [Q&A / Help]
Hi there,

I find the Sankey chart too limited for our use cases. 
When I've looked at the eChart library, I almost found what I need  : [Sankey charts](https://echarts.apache.org/examples/en/editor.html?c=sankey-simple)

I'm a data analyst, not a Python developer (meaning I'm not familiar with docker, builds, etc.).

How should I proceed to implement a new funnel chart using eChart library ?

Thank you for your help.

**Comment by mecca1991** [comment]
Hi @ValentinC-BR, 
You can check out this article from the following link: https://preset.io/blog/building-custom-viz-plugins-in-superset-v2/ on how to get started building custom visualization plugin. I  think you should have some programming knowledge of TypeScript in order to implement this.
